contrib/terraform/openstack/group_vars/all.yml

```diff
@@ -105,21 +105,18 @@ kube_apiserver_insecure_port: 8080 # (http)
 # into appropriate IP addresses. It's highly advisable to run such DNS server,
 # as it greatly simplifies configuration of your applications - you can use
 # service names instead of magic environment variables.
-# You still must manually configure all your containers to use this DNS server,
-# Kubernetes won't do this for you (yet).
 
-# Do not install additional dnsmasq
-skip_dnsmasq: false
```

Comment:
This possible with:
```yaml
# Can be dnsmasq_kubedns, kubedns or none
dns_mode: dnsmasq_kubedns

# Can be docker_dns, host_resolvconf or none
resolvconf_mode: docker_dns
```

I decided to remove the skip_dnsmasq and setup_dns flags, as I always got confused by them. skip_dnsmasq had to be 'false' to enable dnsmasq and setup_dns had to be 'true' to enable it. I thought this was not consistent and may confuse people about what is possible and what not.

I hope with the 2 new modes, it's clearer that the dns mode (dnsmasq+kubedns or kubedns) and resolv conf related stuff are actually 2 different things. This is something that caused a lot of confusion on my side as you probably already have noticed in our discussions ;)